engines/billing/questions.md

```diff
@@ -0,0 +1,22 @@
+Questions
+====================
+
+1. Should admin be able to see subscriptions?
+
+2. Should user be able to upgrade or downgrade subscriptions?
+
+3. Should confirmations emails be allowed?
+
+See here:
+https://articles.braintreepayments.com/guides/recurring-billing/email-notifications
+https://articles.braintreepayments.com/control-panel/transactions/email-receipts
+
+4. Braintree uses JS for dropin. I'm not sure but this may not work in browsers
+such as Tor. Is there a problem?
+JS can be improved and changed to 'Hosted Fields'. See here:
+https://developers.braintreepayments.com/javascript+ruby/guides/hosted-fields/overview
+
+5. There is another payment method that can be allowed: SEPA Direct Debit (on
```

Comment:
Yep! Only problem is that it is still on beta. It is in a limited beta and parts of the integration may change. It is currently only open to merchants based in Europe and limited to German customers. See here: https://developers.braintreepayments.com/ios+ruby/guides/sepa/overview